Hymns/Display/HymnLyricsView.swift

```diff
@@ -17,21 +17,29 @@ public struct HymnLyricsView: View {
         guard !lyrics.isEmpty else {
             return AnyView(Text("loading..."))
         }
+        var verseIndex: Int = 0
 
+        func isVerse(verseType: VerseType) -> Int {
+            if verseType != VerseType(rawValue: "chorus") {
+                verseIndex += 1
+                return verseIndex
+            }
+            return 0
+        }
```

Comment:
I'd prefer to move this logic to a view model. So instead of the HymnLyricsViewModel having a list of Verse objects, we can make it have a VerseViewModel object that contains the verse number as well.

Reply:
Agreed, makes sense. Let me think about it and see what I can do.

Reply:
I moved the logic to a VerseViewModel struct like you mentioned above. It looks cleaner and builds. Please review when you can so I can merge to master.